android/quest/src/main/java/org/smartregister/fhircore/quest/ui/profile/ProfileViewModel.kt

```diff
@@ -246,10 +247,18 @@ constructor(
         }
       }
       is ProfileEvent.OnChangeManagingEntity -> {
+        val relationshipCode =
+          Code().apply {
+            system = "http://hl7.org/fhir/ValueSet/relatedperson-relationshiptype"
+            code = "99990006"
+            display = "Family Head"
```

Comment:
where should I move these Constants? @ellykits

Reply:
the display can go the translation strings file. I wonder if we can move the entire code object to a constants file instead of the parts? Seems like that make sense, since the code value should always go w/the system value

Reply:
Through [ManagingEntityConfig](https://github.com/opensrp/fhircore/blob/main/android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/profile/ManagingEntityConfig.kt)

Reply:
I also think the implementation for changing the managing entity should be generic enough to support other ResourceTypes that can also be managing entities like [Organization, Practitioner and PractitionerRole](http://hl7.org/fhir/R4/group-definitions.html#Group.managingEntity) resources.

Reply:
We don't have to support other ManagingEntity resource types now, however a simple conditional check if member resource type == Patient, use RelatedResource will suffice.

Reply:
The configuration plans sound good

Reply:
I've resolved one feedback related to relationship code done through the configuration, The other suggestion related to have a conditional check on resourceType, we need to discuss this. @ellykits

Reply:
Resolved feedback after discussing with @ellykits